meta-facebook/yv35-bb/CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ project(yv35-bb)
 
 set(common_path ../../common)
 FILE(GLOB app_sources src/ipmi/*.c src/platform/*.c src/lib/*.c )
-FILE(GLOB common_sources ${common_path}/*.c ${common_path}/ipmi/*.c ${common_path}/host/*.c ${common_path}/sensor/*.c ${common_path}/usb/*.c ${common_path}/hal/*.c ${common_path}/dev/*.c ${common_path}/lib/*.c ${common_path}/ipmb/*.c)
+FILE(GLOB common_sources ${common_path}/service/*.c ${common_path}/service/*/*.c ${common_path}/hal/*.c ${common_path}/dev/*.c ${common_path}/lib/*.c ${common_path}/shell/*.c)
```

Comment:
I think we still need these 2 to build?
```
${common_path}/pldm/*.c 
${common_path}/mctp/*.c
```

Reply:
We have fixed this and do the forced update.
Please review it. Thanks.